src/WebJobs.Script/Binding/Http/HttpTriggerAttributeBindingProvider.cs

```diff
@@ -79,6 +80,11 @@ public HttpTriggerBinding(HttpTriggerAttribute attribute, ParameterInfo paramete
                     }
                 }
 
+                if(attribute.IsProxy)
```

Comment:
could the if statement be changed to a single assignment statement "_isProxy = IsProxy"?

---
In reply to: [102524457](https://github.com/Azure/azure-webjobs-sdk-script/pull/1224#discussion_r102524457) [](ancestors = 102524457)